Drop after devel/glib20 is updated to 2.58 or later.

../tests/test-lfb-integration.c:26:17: warning: implicit declaration of function 'g_canonicalize_filename' is invalid in C99 [-Wimplicit-function-declaration]
  servicesdir = g_canonicalize_filename (relative, NULL);
                ^
ld: error: undefined symbol: g_canonicalize_filename
>>> referenced by test-lfb-integration.c
>>>               tests/test-lfb-integration.p/test-lfb-integration.c.o:(fixture_setup)

--- tests/test-lfb-integration.c.orig	2020-07-26 11:06:35 UTC
+++ tests/test-lfb-integration.c
@@ -23,7 +23,11 @@ fixture_setup (TestFixture *fixture, gconstpointer unu
 
   fixture->dbus = g_test_dbus_new (G_TEST_DBUS_NONE);
   relative = g_test_build_filename (G_TEST_BUILT, "services", NULL);
+#if GLIB_CHECK_VERSION(2, 58, 0)
   servicesdir = g_canonicalize_filename (relative, NULL);
+#else
+  servicesdir = realpath (relative, NULL);
+#endif
   g_free (relative);
 
   g_test_dbus_add_service_dir (fixture->dbus, servicesdir);
